resilience.markdown

```diff
@@ -14,40 +16,39 @@ Given Turing's commitment to advancing diversity in the field, resilience will a
 ## Goal of Session
 Identify personal strategies for developing resilience and articulate ways we can support each other in this work at Turing. 
 
-## Read (15 min)
-<a href="https://www.youtube.com/watch?v=wMMGhH_cR-8" target="_blank">![https://www.youtube.com/watch?v=wMMGhH_cR-8](./images/apollo_13.png)</a>
-
-Read the *New Yorker* and *Inc* articles below. Pay attention to how the authors discuss 1) the connection between individuals' mindsets and resilience, and 2) how resilience is a *skill*.
-
-#### Required Reading
-* ["How People Learn to Become Resilient"](http://www.newyorker.com/science/maria-konnikova/the-secret-formula-for-resilience) (*The New Yorker*, February 11, 2016)
-* ["What The New York Times, New Yorker and Most of Academia Got Wrong About Resilience"](http://www.inc.com/joshua-spodek/resilience-what-the-new-york-times-new-yorker-and-most-of-academia-got-wrong.html) (*Inc*, March 31, 2016)
-
-#### Additional Reading
-* ["The Profound Emptiness of Resilience"](http://www.nytimes.com/2015/12/06/magazine/the-profound-emptiness-of-resilience.html) (*The New York Times*, December 6, 2015, note: paywall)
+## Introductions (10 min)
+- 2 Truths and a Lie
```

Comment:
I like this!  Sometimes people have a hard time thinking through this, so you might want to give them a heads up on Friday morning.